linux_sql/README.md

```diff
@@ -0,0 +1,43 @@
+# 1) Introduction #
+Cluster Monitor Agent is an internal tool that monitors the cluster resources. It manages servers connected 
+internally via switch and records the hardware specifications and storage details of the servers and stores 
+them into a database via PostgreSQL. The infrastructure team is then able to query information regarding server 
+host id by CPU number and sort by their memory size, average memory usage of each host and display when servers 
+are down.
+
+# 2) Architecture and Design #
+![my_architecture](./assets/linuxSqlArchitecture.jpg)
+
+# 3) Explanation of files #
+###### 1) ./scripts/host_info.sh ######
```

Comment:
###### is a title
please use either inline code or block code format in this section

Reply:
markdown board `**bold**`

Reply:
Edited with inline code